test/runtests.sh

```diff
@@ -13,7 +13,11 @@ fi
 javac  ../src/seedu/addressbook/AddressBook.java -d ../bin
 
 # run the program, feed commands from input.txt file and redirect the output to the actual.txt
-java -classpath ../bin seedu.addressbook.AddressBook < input.txt > actual.txt
+java -classpath ../bin seedu.addressbook.AddressBook " " < /dev/null > actual.txt
+java -classpath ../bin seedu.addressbook.AddressBook 'directoryThatDoesNotExist/valid.filename' < /dev/null >> actual.txt
+java -classpath ../bin seedu.addressbook.AddressBook '.noFilename' < /dev/null >> actual.txt
+java -classpath ../bin seedu.addressbook.AddressBook 'invalid.filename?' < /dev/null >> actual.txt
```

Comment:
`?` is a valid filename character on Linux. The only test case we can use is to have a NUL byte in the filename, but unfortunately that's impossible when we are passing them through command arguments since command-line arguments are C strings which are NUL-terminated.

So maybe we should just drop this test, on Linux and on Windows as well.